packages/cozy-harvest-lib/src/components/TriggerManager.jsx

```diff
@@ -183,23 +183,98 @@ export class TriggerManager extends Component {
 }
 
 TriggerManager.propTypes = {
+  /**
+   * Account document. Used to get intial form values.
+   * If no account is passed, AccountForm will use empty initial values.
+   * @type {Object}
+   */
   account: PropTypes.object,
+  /**
+   * Konnector document. AccountForm will check the `fields` object to compute
+   * fields.
+   * @type {Object}
+   */
   konnector: PropTypes.object.isRequired,
+  /**
+   * Indicates if the TriggerManager has to show errors. Sometimes errors may be
+   * displayed elsewhere. However, a KonnectorJobError corresponding to a login
+   * error is always displayed. Transmitted to AccountForm.
+   * @type {Boolean}
+   */
   showError: PropTypes.bool,
+  /**
+   * Existing trigger document to manage.
+   * @type {Object}
+   */
   trigger: PropTypes.object,
+  /**
+   * Indicates if the given trigger is already running, i.e. if it has been
+   * launched and if an associated job with status 'running' exists.
+   * @type {[type]}
+   */
   running: PropTypes.bool,
+  /**
+   * Translation function
+   */
+  t: PropTypes.func,
+  //
   // mutations
+  //
+  /**
+   * Permission mutation
+   * @type {Func}
+   */
   addPermission: PropTypes.func,
+  /**
+   * File mutation
+   * @type {Func}
+   */
   addReferencesTo: PropTypes.func,
+  /**
+   * Trigger mutation
+   * @type {Func}
+   */
   createTrigger: PropTypes.func.isRequired,
+  /**
+   * Trigger mutations
+   * @type {Func}
+   */
   createDirectoryByPath: PropTypes.func,
+  /**
+   * Trigger mutation
+   * @type {Func}
+   */
   launchTrigger: PropTypes.func.isRequired,
+  /**
+   * Account mutation
+   * @type {Func}
+   */
   saveAccount: PropTypes.func.isRequired,
+  /**
+   * Trigger mutations
+   * @type {Func}
+   */
   statDirectoryByPath: PropTypes.func,
+  /**
+   * Job mutations
+   * @type {Func}
+   */
   waitForLoginSuccess: PropTypes.func.isRequired,
-  // hooks
+  //
+  // Callbacks
+  //
+  /**
+   * Callback invoked when the trigger has been launched and that the login
+   * to the remote service has succeed.
+   * @type {Func}
+   */
   onLoginSuccess: PropTypes.func,
-  onSuccess: PropTypes.func
+  /**
+   * Callback invoked when the trigger has been launched and the job ended.
+   * Can be called before `onLoginSucces` for some konnectors.
```

Comment:
"onLoginSuccess". I don't understand this comment. Is it something I must pay attention to ?